src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -1074,6 +1074,54 @@ impl ToQemuParams for DeviceVirtconsole {
     }
 }
 
+// RTC represents a qemu Real Time Clock configuration.
+#[derive(Debug)]
+struct Rtc {
+    // Base is the RTC start time.
+    base: String,
+
+    // Clock is the is the RTC clock driver.
+    clock: String,
+
+    // DriftFix is the drift fixing mechanism.
+    driftfix: String,
+}
+
+impl Rtc {
+    fn new(base: &str, clock: &str, driftfix: &str) -> Rtc {
```

Comment:
RTC isn't user configurable. The command line we want is exactly `-rtc base=utc,driftfix=slew,clock=host`.
I suggest you drop the arguments to this functions and you just hard-code the values in the `Rtc` instance below.